guide/README.md

```diff
@@ -369,6 +370,27 @@ block `Mi` is lowered by one half step; on the right, both *Pitch*
 blocks are raised by one half step. (You can also use a double-sharp
 or double-flat accidental.)
 
+
+#### <a name= "DOUBLE-SHARPS-AND-FLATS">3.2.2.1 Double Sharps And Flats</a>
+
+Double sharps and flats, although not very common, are used in music
+notation to accurately represent pitches that are altered by two
+semitones. They arise in compositions that explore chromaticism,
+modulations to distant keys, or non-traditional scales where such
+alterations are necessary for harmonic or melodic purposes. Double
+sharps and flats help maintain the correct intervals, ensuring
+precise notation and clarity of musical expression.
+
+![double sharp and flat](./transform1_1.png "Using Double Sharp and Flat blocks")
+
+![la campanella](./transform1_2.png "La Campanella - Liszt")
+
+![mikrokosmos](./transform1_3.png "Mikrokosmos V Nr. 131 - Bartók")
+
+The example on the left shows how to play the double sharp as played in
```

Comment:
In the text, it refers to examples on the left and right. However, they display for me above and below. Perhaps just refer to them as first and second example, or, if you can, add captions underneath and refer to them as "the example by Liszt" and "the example by Bartok".

![Screenshot from 2024-04-22 16-02-37](https://github.com/sugarlabs/musicblocks/assets/13454579/23f8be32-d031-4615-8778-d11153081501)